docs/accessories/developer/rendering_api.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Rendering API
+project: accessories
+---
+
+To link a given item to a specific `AccessoryRenderer` you will need to register such within the `AccessoriesRendererRegistry` class. By **default** all accessory items that have no renderer registered will get the `DefaultAccessoryRenderer` which attempts to render the block/item on the player in a position where the slot is located on the player. 
+
+!!! info "Default Accessory Rendering Behavior"
+    All default slots have implemented renderers for such but new slots will need to implement there own using the `DefaultAccessoryRenderer#registerHelper` method. More about the `DefaultAccessoryRenderer` can be found below.
+
+If you need a simple item rendering for your accessory than you could look at `SimpleAccessoryRenderer` as all the is required of you is to override the `align` method to transform the rendering to specific location on the player.
```

Comment:
Possible grammar mistake? "as all *the* is required" -> "as all *that* is required"